resources/assets/js/components/CreateProject.js

```diff
@@ -0,0 +1,120 @@
+import React, {Component} from 'react';
+import {browserHistory} from 'react-router';
+import MyGlobleSetting from './MyGlobleSetting';
+
+
+class CreateProject extends Component {
+  constructor(props){
+    super(props);
+    this.state = {name: '', information: '', deadline: '', type: '', status: ''};
+    this.handleChange1 = this.handleChange1.bind(this);
```

Comment:
Why do you have to use bind(this) here ?

Reply:
i bind to get values this of input